actions/platform/tam/setup_actions.py

```diff
@@ -0,0 +1,18 @@
+from typing import Any, Text, Dict, List
+
+from rasa_sdk import Action, Tracker
+from rasa_sdk.executor import CollectingDispatcher
+from rasa_sdk.events import SlotSet
+
+from actions.platform.tam import _TAM_SLOTS
+
+
+class ActionAccessRequestTAMReset(Action):
+    def name(self) -> Text:
+        return "action_access_request_tam_reset"
+
+    async def run(
+        self, dispatcher: CollectingDispatcher, tracker: Tracker, domain: Dict
+    ) -> List[Dict[Text, Any]]:
+        print("Resetting TAM slots")
```

Comment:
I suppose this is debug, lets remove or add more info :)

Reply:
Yup, it's gone